authorization/role/service/role_management_service.go

```diff
@@ -151,3 +152,27 @@ func (s *roleManagementServiceImpl) Assign(ctx context.Context, assignedBy uuid.
 
 	return err
 }
+
+// AssignAsAdmin assigns an identity (users, organizations, teams or groups) with a role for a specific resource.
+// This method doesn't check any permissions and assumes that the caller does all needed permissions checks.
+// As an example: this method is to be used when creating a resource (space) to assign initial admin role to the resource creator.
+// IMPORTANT: This is a transactional method, which manages its own transaction/s internally
+func (s *roleManagementServiceImpl) AssignAsAdmin(ctx context.Context, assignedTo uuid.UUID, roleName string, res resource.Resource) error {
```

Comment:
I find this method name a little confusing, as there is no actual "admin" role as described, however there are resource admin roles.  Could we perhaps call it something like `ForceAssignNamedRole()` instead?

Reply:
Yeah.. I don't like this name either. Was using it as a temporal name until I come up with a better name but I didn't ;)
I like ForceAssignNamedRole or just ForceAssign.

Reply:
Done in https://github.com/fabric8-services/fabric8-auth/pull/498/commits/a82bf3fd432bb4f3f5f7307f60d073c1a1cc3716